public/app/core/components/GrotNotFound/GrotNotFound.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import SVG from 'react-inlinesvg';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+import notFoundSvg from '../../../../img/grot-not-found.svg';
+
+import useMousePosition from './useMousePosition';
+
+export interface Props {
+  width?: number;
+  height?: number;
+}
+
+export const GrotNotFound = ({ width, height }: Props) => {
+  const { x, y } = useMousePosition();
+  const styles = useStyles2(getStyles, x, y);
+  return <SVG src={notFoundSvg} className={styles.svg} height={height} width={width} />;
+};
+
+GrotNotFound.displayName = 'GrotNotFound';
+
+const getStyles = (theme: GrafanaTheme2, xPos: number | null, yPos: number | null) => {
+  const { innerWidth, innerHeight } = window;
+  const heightRatio = yPos && yPos / innerHeight;
+  const widthRatio = xPos && xPos / innerWidth;
+  const rotation = heightRatio !== null ? getIntermediateValue(heightRatio, -20, 5) : 0;
```

Comment:
I guess you eyeballed the start and end values?

Reply:
yup, i've extracted them out into constants now as well 👍